app/assets/stylesheets/config/_fonts.scss

```diff
@@ -3,11 +3,43 @@ $body-font: 'Open Sans', sans-serif;
 $headers-font: 'Open Sans', sans-serif;
 
 // To use a font file (.woff) uncomment following lines
-// @font-face {
```

Comment:
Loading custom fonts locally in order to avoid to avoid render blocking.
Resources: https://blog.logrocket.com/5-tricks-to-eliminate-render-blocking-resources/